setup.py

```diff
@@ -96,7 +96,10 @@ def get_extensions():
     sources = [os.path.join('erfa', 'ufunc.c')]
     include_dirs = []
     libraries = []
-    define_macros = []
+    define_macros = [
```

Comment:
Why this change? It seems reasonable, but perhaps good to add why it is even needed?